api/src/main/java/openconsensus/stats/StatsRecorder.java

```diff
@@ -54,5 +67,5 @@
    *     measurements are associated with.
    * @since 0.1.0
    */
-  public abstract void record(List<Measurement> measurements, TagMap tags, SpanContext spanContext);
+  void record(List<Measurement> measurements, TagMap tags, SpanContext spanContext);
```

Comment:
I have some concerns about adding a dependency on Tracing from Stats - I know that it's at the API level only so it's less of a concern but is seems like you could get cyclic dependencies if you're not careful.